package.json

```diff
@@ -13,14 +13,21 @@
     "eject": "react-native-scripts eject",
     "android": "react-native-scripts android",
     "ios": "react-native-scripts ios",
-    "test": "jest"
+    "ci": "echo hello world, this is a stub",
+    "test": "jest --maxWorkers=4",
+    "test:coverage": "jest --coverage",
+    "lint": "eslint ."
   },
   "jest": {
     "preset": "jest-expo"
   },
   "dependencies": {
+    "eslint": "^3.17.1",
```

Comment:
This is very old - `eslint` is now over version 5 now. Using the npm VS Code extension, you can hover over this to see the latest version.